### PR TITLE
Fix recursive call in TwoFactorWebAuthnDeleteRequestModel

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change
- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

Resolves a recursive call when attempting to delete a WebAuthn token.

## Before you submit
- [ ] If making database changes - I have also updated Entity Framework queries and/or migrations
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

**Review by MGibson1 — DISMISSED** [review]
Please confirm fix locally before notifying devops to update servers

File: src/Core/Models/Api/Request/TwoFactorRequestModels.cs [hunk 1]
```diff
@@ -238,7 +238,7 @@ public class TwoFactorWebAuthnDeleteRequestModel : SecretVerificationRequestMode
 
         public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
         {
-            foreach (var validationResult in Validate(validationContext))
+            foreach (var validationResult in base.Validate(validationContext))
```
Comment (kspearrin) [inline src/Core/Models/Api/Request/TwoFactorRequestModels.cs:241]: does this need to override base class? I am seeing warnings in VS for me.

Comment (Hinton) [reply src/Core/Models/Api/Request/TwoFactorRequestModels.cs:241]: I'm not familiar with the particular warning, but it wants me to add `new` before IEnumerable.

Comment (kspearrin) [reply src/Core/Models/Api/Request/TwoFactorRequestModels.cs:241]: ok, `new` it is. Try that and verify it works still I guess.

Comment (MGibson1) [reply src/Core/Models/Api/Request/TwoFactorRequestModels.cs:241]: new will shadow the previous implementation, but I believe will only work if the type is specifically cast to `TwoFactorRequestModel`, I don't think that's what we want

Comment (MGibson1) [reply src/Core/Models/Api/Request/TwoFactorRequestModels.cs:241]: more than likely it'd be better to make the base class `virtual` and `override` here

**Review by kspearrin — APPROVED** [review]

**Pull request merged**